src/main/java/seedu/storemando/commons/core/Messages.java

```diff
@@ -10,4 +10,5 @@
     public static final String MESSAGE_INVALID_ITEM_DISPLAYED_INDEX = "The item index provided is invalid";
     public static final String MESSAGE_MORE_THAN_ONE_ITEM_LISTED_OVERVIEW = "%1$d items listed!";
     public static final String MESSAGE_LESS_THAN_TWO_ITEMS_LISTED_OVERVIEW = "%1$d item listed!";
+    public static final String MESSAGE_NUMBER_OF_ITEMS_LISTED_OVERVIEW = "%1$d item listed!";
```

Comment:
I think u can use the above two lines to print the number of items listed. One has the s in items while the other doesn't to differentiate between single and multiple items being returned.

Reply:
Agree with kums!

Reply:
I agree that the message is the same as the above two lines. But, the description doesn't feed my usage, that's why I added a new string. I shall change to a different string description.